app/code/Magento/Checkout/view/frontend/web/js/proceed-to-checkout.js

```diff
@@ -21,9 +21,10 @@ define([
                 authenticationPopup.showModal();
 
                 return false;
+            }else{
```

Comment:
Please, make sure you keep the proper code style (https://public-results-storage-prod.magento-testing-service.engineering/reports/magento/magento2/pull/27101/1d3682240ab21db5a753597a32170b2d/Statics/allure-report-b2b/index.html#suites/3a29d43e5c1686787ff8c2a77e590abb/fe7c32f4bafba18c/)